tests/test_reweighting.py

```diff
@@ -137,7 +137,7 @@ def test_mixture_reweighting_1d():
     # TI
     vec_du_dl = vmap(grad(u_fxn, 1), (0, None, None))
     mean_du_dls = jnp.array([jnp.mean(vec_du_dl(traj, lam, ref_params)) for (traj, lam) in zip(trajs, lambdas)])
```

Comment:
did we want these to be numpy as well?

Reply:
sure -- the inputs are device arrays from jax.vmap (and the usages on the subsequent lines are jax), but this can be converted to original numpy